pkg/helpers/imgworker/unpack.go

```diff
@@ -20,6 +21,7 @@ import (
 
 // Unpack exports an image to a rootfs destination directory.
 func (c *Client) Unpack(image, dest string) error {
+	_,_ = bus.Manager.Publish(bus.EventImagePreUnPack, c)
```

Comment:
Maybe instead of the client, it would be better to publish `image` and `dest` also ?

Reply:
done!